install_config/install/advanced_install.adoc

```diff
@@ -589,6 +595,84 @@ openshift_master_overwrite_named_certificates=true
 ----
 ====
 
+[[advanced-install-cluster-metrics]]
+== Configuring Cluster Metrics
+
+Cluster metrics are not set to automatically deploy by default. Set the
+following to enable cluster metrics when using the advanced install:
+
+====
+----
+[OSEv3:vars]
+
+openshift_hosted_metrics_deploy=true
+----
+====
+
+=== Metrics Storage
+
+The `*openshift_hosted_metrics_storage_kind*` variable must be set in order to
+use persistent storage. If `*openshift_hosted_metrics_storage_kind*` is not set,
+then cluster metrics data is be stored in an `EmptyDir` volume, which will
```

Comment:
"then cluster metrics data is be stored..." 
Whoops. I think you'll need to rewrite that.